### DISCUSSION TITLE
2.8.4 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#4527] [讨论]
· 下载 Mod 时会使用 MCIM 国内镜像源（#4267，@z0z0r4，PR @youzi-2333）
　　自动安装也会从镜像源获取 Fabric API 信息（#4327，@ZhangJinyuJerry）
　　将 z0z0r4 添加到特别鸣谢列表
· 打开 PCL 时会自动安装同目录下的 modpack.zip（#3066，@FortyTwoCn，PR @youzi-2333）
· 所有 4 月 1 日发布的版本会被自动视为愚人节版（#3945、#4449，PR @Silverteal、@tangge233）
· 爱发电域名迁移至 afdian.com（#4271，@SuoHouDaiShi）
· 会在日志中输出详细的 Windows 版本（#4351，@Pigeon0v0）
· 添加 MCC 15 周年披风的翻译（#4366，@Hill23333，PR @allMagicNB）
· 资源下载添加 1.21.1 筛选项（#4441，PR @3gf8jv4dv）
· 不再建议使用离线皮肤功能（#4500，@Silverteal）
· 若整合包提供了有效的版本名，安装时就不再要求输入版本名
· 若使用正式版，则不在设置显示复制识别码、输入解锁码按钮
· 更新 Mod 数据库
· 更新 7 月赞助者列表，感谢大家的支持！
· 更新帮助库
　　替换帮助库中失效的图片（LTCatt/PCL2Help#372，@z0z0r4，PR @WorldHim）
　　添加“多人游戏已被禁用”的更详细解决方案（LTCatt/PCL2Help#385，PR @Enigma-Soul）
　　追加对自动安装 modpack.zip 的功能介绍
　　编修（LTCatt/PCL2Help#383，PR @WTP016-CN）
· 修复了以下 18 个 Bug
　　[严重] 1.20.1+ 离线登录使用正版皮肤时无法保存游戏（#3746，@jdzjjjhq）
　　[严重] 安装的 1.14~1.15 Forge+OptiFine 无法进入世界（#4134，@xiaoj-ui）
　　有多个下载任务时有小概率出错（#3771，@Casper233）
　　从回收站拖拽安装禁用的 Mod 后扩展名有误（#4227，@Ad-closeNN）
　　可能选择错误的 Java 8（#4261，@al-wei）
　　可能错误地将官启 Java 视作强制指定的 Java（#4286，@Pigeon0v0）
　　MyIconButton 无法设置为红色主题（#4290，@litwak913）
　　删除 launcher_profiles.json 中的令牌（#4313，@cucumbertw）
　　Mod 管理页的下边栏更新按钮不会刷新（#4377，@XeF2i）
　　第三方登录下重新登录后错误的使用了缓存（#4382，@wuliaodexiaoluo）
　　ARM 架构的 CPU 可能无法启动 MC（#4396，@Sunch233）
　　部分映射表文件无法使用镜像源下载（#4425，@t57282）
　　整合包的 [烹饪] 分类应译为 [大杂烩]（#4435，@Chiloven945，PR @3gf8jv4dv）
　　统一通行证所需 Java 版本不准确（#4446，@CrazyNutLz）
　　崩溃报告中，游戏崩溃前的输出可能有编码问题（#4482，PR @allMagicNB）
　　错误报告导出的启动脚本编码有误（#4488，@wuliaodexiaoluo）
　　无法使用镜像源获取 NeoForge 版本列表（#4491，@LxHTT）
　　Mod 下载列表加载失败时无法点击重试

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.8.4'>2.8.4</a>.</em>

**Comment by 3gf8jv4dv** [comment]
> · 资源下载添加 1.12.1 筛选项（https://github.com/Hex-Dragon/PCL2/pull/4441 ，PR @3gf8jv4dv）

```diff
- · 资源下载添加 1.12.1 筛选项（https://github.com/Hex-Dragon/PCL2/pull/4441，PR @3gf8jv4dv）
+ · 资源下载添加 1.21.1 筛选项（https://github.com/Hex-Dragon/PCL2/pull/4441，PR @3gf8jv4dv）
```

**Reply by LTCatt** [reply]
修改了
